static/js/containers/UserPage_test.js

```diff
@@ -50,247 +38,288 @@ describe("UserPage", () => {
     ));
   };
 
-  afterEach(() => {
-    helper.cleanup();
-  });
+  describe ("Authenticated user page", () => {
+    beforeEach(() => {
+      helper = new IntegrationTestHelper();
+      listenForActions = helper.listenForActions.bind(helper);
+      renderComponent = helper.renderComponent.bind(helper);
 
-  describe("Education History", () => {
-    let deleteButton = div => {
-      return div.getElementsByClassName('profile-tab-card')[1].
-        getElementsByClassName('delete-button')[0];
-    };
-
-    it('shows the education component', done => {
-      renderComponent(`/users/${SETTINGS.username}`, userActions).then(([, div]) => {
-        let title = div.getElementsByClassName('profile-card-title')[1];
-        assert.equal(title.textContent, 'High school');
-        done();
-      });
+      patchUserProfileStub = helper.sandbox.stub(api, 'patchUserProfile');
+
+      helper.profileGetStub.returns(
+        Promise.resolve(Object.assign({}, USER_PROFILE_RESPONSE, {
+          username: SETTINGS.username
```

Comment:
This will update the username in the profile but we will still be using whatever username was in the url to store it in `profiles` object. Can you change `helper.profileGetStub` here and in the unauthenticated version to use `withArgs` to make sure we have the correct username in the URL too?